### PR TITLE
Fix: Add shortcut to rotate camera by 90degrees #14

### PR DESCRIPTION
**Pull request by dk1242**
I have added a shortcut for rotating the camera by +/-90 degrees.
Pressing the key "4" will rotate the camera by -90 degrees and "6" will rotate it by 90 degrees.

Fixes for #14

### CODE REVIEW

File: library/public/camera.h [hunk 1]
```diff
@@ -44,6 +44,7 @@ class F3D_EXPORT camera
   virtual vector3_t getViewUp() = 0;
   virtual void getViewUp(vector3_t& up) = 0;
   virtual camera& setViewAngle(const angle_deg_t& angle) = 0;
+  virtual camera& rotateCamera(const angle_deg_t& angle) = 0;
```
Comment (mwestphal) [inline library/public/camera.h:47]: I think you can use `roll` instead of adding a method

File: library/src/interactor_impl.cxx [hunk 2]
```diff
@@ -323,6 +323,10 @@ class interactor_impl::internals
         self->SetViewOrbit(ViewType::VT_FRONT, self);
         render = true;
         break;
+      case '2':
```
Comment (mwestphal) [inline library/src/interactor_impl.cxx:326]: lets use 4 and 6 instead :)

Comment (dk1242) [reply library/src/interactor_impl.cxx:326]: Okay but please confirm, 4 for -90 degree rotation and 6 for +90 degree rotation?

File: library/public/camera.h (hunk 1 above)
Comment (dk1242) [reply library/public/camera.h:47]: yeah much better

File: library/src/interactor_impl.cxx (hunk 2 above)
Comment (mwestphal) [reply library/src/interactor_impl.cxx]: yes

**Review by mwestphal — CHANGES_REQUESTED** [review]
looks nice! Please also add doc for this feature in:

 - vtkF3DRenderer::ConfigureCheatSheet
 - doc/user/INTERACTIONS.md

File: doc/user/INTERACTIONS.md [hunk 3]
```diff
@@ -57,7 +57,9 @@ Note that some hotkeys can be available or not depending on the file being loade
 Camera Hotkeys:
 * <kbd>1</kbd>: front view camera.
 * <kbd>3</kbd>: right view camera.
+* <kbd>4</kbd>: rotate camera by -90 degrees.
```
Comment (mwestphal) [inline doc/user/INTERACTIONS.md:60]: roll the camera left by 90 degrees

File: doc/user/INTERACTIONS.md [hunk 4]
```diff
@@ -57,7 +57,9 @@ Note that some hotkeys can be available or not depending on the file being loade
...
 * <kbd>5</kbd>: toggle orthographic projection.
+* <kbd>6</kbd>: rotate camera by 90 degrees.
```
Comment (mwestphal) [inline doc/user/INTERACTIONS.md:62]: roll the camera right by 90 degrees

**Review by mwestphal — APPROVED** [review]

**Pull request merged**